configure.ac

```diff
@@ -5822,13 +5823,20 @@ AC_ARG_WITH(
     AS_CASE([$with_readline],
       [editline|edit], [with_readline=edit],
       [yes|readline], [with_readline=readline],
-      [no], [],
+      [no], [with_readline=default],
```

Comment:
This does not feel right; if `--with-readline=no` we ... default to system readline?

Reply:
Ah it was my mistake.